Content.Shared/_Sunrise/SunriseCCVars/SunriseCCVars.cs

```diff
@@ -39,6 +39,12 @@ public sealed partial class SunriseCCVars : CVars
     public static readonly CVarDef<bool> TTSClientEnabled =
         CVarDef.Create("tts.client_enabled", true, CVar.CLIENTONLY | CVar.ARCHIVE);
 
+    /// <summary>
+    /// Option to disable TTS queue in radio for client
+    /// </summary>
+    public static readonly CVarDef<bool> TTSClientQueueEnabled =
+        CVarDef.Create("tts.client.queue_enabled", true, CVar.CLIENTONLY | CVar.ARCHIVE);
```

Comment:
```suggestion
        CVarDef.Create("tts.queue_enabled", true, CVar.CLIENTONLY | CVar.ARCHIVE);
```

Эти точки ипользуются для разделения сиваров по категориям в конфиге. Я не уверен, что оно сработает для двух точек. Поэтому лучше так